<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>733</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 36, 76);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelProdutos_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>   Produto</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>588</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSair">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(20, 36, 76);
color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>731</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 234, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>611</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAplic">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(46, 122, 107);
color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Aplicar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>731</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelInfoContBal">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Descrição</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInfoLote">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Lote</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditQtd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 10px;
border:2px solid rgb(204, 204, 204);
border-radius:4px;</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCodProdut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 10px;
border:2px solid rgb(204, 204, 204);
border-radius:4px;</string>
     </property>
     <property name="maxLength">
      <number>14</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelInfoQtd">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Quantidade</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDesc">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 10px;
border:2px solid rgb(204, 204, 204);
border-radius:4px;
background-color: rgb(238, 238, 238);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInfoCodProdut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Produto</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInfoVenc">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Vencimento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLote">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 10px;
border:2px solid rgb(204, 204, 204);
border-radius:4px;</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEditVenc">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 5px;</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditCodProdut</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEditQtd</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditQtd</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEditLote</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAplic</sender>
   <signal>pressed()</signal>
   <receiver>lineEditCodProdut</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditLote</sender>
   <signal>returnPressed()</signal>
   <receiver>dateEditVenc</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditVenc</sender>
   <signal>editingFinished()</signal>
   <receiver>btnAplic</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
